performant-software/textlab
# Fixes step permissions on shared sequences

Note: Most of this PR has already been tested and approved (sorry I set it to review before I'd pushed this other check) please check one more thing and we'll close it - Verify that you cannot alter the order of leaves on a shared doc.

Thanks!

**What does this PR Do**
#closes 187
#closes 151
Fixes the permission on leaf reorder
Fixes the permissions on shared sequences

**How to Test**
Go here: https://mel-textlab-pr-188.herokuapp.com
Verify that you cannot change the order of leaves in shared projects by dragging them around the tree 
Verify that step order cannot be changed on shared sequences
